app/views/articles/show.html.erb

```diff
@@ -203,12 +202,31 @@
     <%= render "articles/full_comment_area" %>
   </div>
 
-# Line 199 ends the div holding the main section of the article
-# Somewhere below...I believe is a "classic dev post" followed by "another post you might like"
-# Depending on where we decide to put these videos would be either right here at the top or maybe below the suggested_articles
+<!-- Line 199 ends the div holding the main section of the article
+Somewhere below...I believe is a "classic dev post" followed by "another post you might like"
+Depending on where we decide to put these videos would be either right here at the top or maybe below the suggested_articles -->
 
   <% cache("article-bottom-content-#{@article.id}", expires_in: 18.hours) do %>
     <% @classic_article = Suggester::Articles::Classic.new(@article, not_ids: [@article.id]).get.first %>
+    <div class="container show-page-content-display">
+      <div class="content-classification">
+        <div class="content-classification-text">Some Videos You Might Like</div>
```

Comment:
Note: Reused CSS which is why this is laid out as nested divs

Reply:
I like reusing the CSS, just shows more so that you were able to read and understand more of the code base and realized that there's already applicable styling within the code base instead of repeating the styling (following DRY concept)